### DISCUSSION TITLE
Logo with name in it

### DISCUSSION THREAD

**Discussion by stripedpajamas** [VSCodium/vscodium#688] [Ideas]
Since this project is one of the primary users of the Open VSX registry, we've been asked to put our logo/name on the adopters page (https://ecdtools.eclipse.org/adopters/). I was going to do this but realized that we don't have a logo that includes the name, and just putting the algae logo won't fly.

If any of the designers who have contributed to this project can make a reasonable logo that includes the project name, I'll send the PR up to Eclipse with it.

cc @hazarek

**Comment by setanarut** [comment]
Do you want a completely new logo? Or will the name be added to the existing one?

**Comment by stripedpajamas** [comment]
Existing one with name

**Comment by setanarut** [comment]
[Untitled.pdf](https://github.com/VSCodium/vscodium/files/6252651/Untitled.pdf)

**Reply by setanarut** [reply]
https://gist.github.com/hazarek/9d093a035e72ace7a35d12dcd02226cc

**Reply by stripedpajamas** [reply]
Thank you @hazarek :heart_eyes:

**Comment by stripedpajamas** [comment]
We're on the adopters page now https://ecdtools.eclipse.org/adopters/
